src/coord.jl

```diff
@@ -133,21 +132,21 @@ end
 #
 function apply_coordinate(coord::Cartesian, aess::Vector{Gadfly.Aesthetics},
                           scales::Dict{Symbol, Gadfly.ScaleElement})
-    pad_categorical_x = Nullable{Bool}()
-    pad_categorical_y = Nullable{Bool}()
+    pad_categorical_x = missing
+    pad_categorical_y = missing
     for aes in aess
-        if !isnull(aes.pad_categorical_x)
-            if isnull(pad_categorical_x)
+        if aes.pad_categorical_x!==missing
```

Comment:
It would be nice to put space around `!==` (and `===` for symmetry) because it's unclear at first glance whether it should be e.g. `x! == y` or `x !== y`.